src/hearings/containers/request-hearing/hearing-timing/hearing-timing.component.spec.ts

```diff
@@ -252,7 +252,7 @@ describe('HearingTimingComponent', () => {
     component.firstDateOfHearingError = null;
     component.firstHearingFormGroup.get('firstHearingDate_day').setValue('25');
     component.firstHearingFormGroup.get('firstHearingDate_month').setValue('10');
-    component.firstHearingFormGroup.get('firstHearingDate_year').setValue(Number(moment().format('YYYY')) + 1);
+    component.firstHearingFormGroup.get('firstHearingDate_year').setValue(Number(moment().format('YYYY')));
```

Comment:
@richardhenryash we have pushed fix for this unit test failure, can you remove this change and rebase with master then it should fix this unit test

Reply:
Will Do